class CPF_Error(Exception):
    pass

class Cliente_Error(Exception):
    pass